client/source/js/modules/model/cost-coverage-ctrl.js

```diff
@@ -301,6 +302,7 @@ define(['./module', 'underscore'], function (module, _) {
      * Retrieve and update graphs based on the current plot models.
      */
     $scope.generateCurves = function () {
```

Comment:
Considering that we already have a property $scope.hasCostCoverResponse, do we really need this one?